src/go/k8s/Makefile

```diff
@@ -96,7 +104,7 @@ docker-build-configurator: prepare-dockerfile
 	docker build --build-arg BUILDPLATFORM='linux/${TARGETARCH}' --build-arg TARGETARCH=${TARGETARCH} --target=configurator -f Dockerfile -t ${CONFIGURATOR_IMG_LATEST} ../
 
 # Preload controller image to kind cluster
-push-to-kind:
+push-to-kind: kind-create
```

Comment:
Should this also have dependency on certmanager-install?

Reply:
🤔 good point, although I'd like to add a check so that we don't download cert manager each time we push an image to kind.